R/pkg/R/column.R

```diff
@@ -132,17 +132,24 @@ createMethods()
 
 #' alias
 #'
-#' Set a new name for a column
+#' Set a new name for an object. Equivalent to SQL "AS" keyword.
```

Comment:
right, this is Scala doc for Column.alias `Gives the column an alias` (which is not very concise)
Dataset.alias `Returns a new Dataset with an alias set.`

I think we need to say `Set a new name to return as a new object` or similar. Actually I think we should say "Column or SparkDataFrame" in place of "object" - what do you think?

I think the `SQL "AS"` part but perhaps it will be more clear if lead with "for Column, ..."?